src/controller/game_controller.py

```diff
@@ -242,39 +220,67 @@ def btn_card(self):
         Manage actions on card button
         """
 
+        card = self.deck.getCard()
+        self.human.add_card(card, self.hand_idx)
+        print("You : " + str(self.human.hands[self.hand_idx].hand))
+        if self.human.hands[self.hand_idx].hand.is_burnt or self.human.hands[self.hand_idx].hand.is_black_jack:
+            print("Is burnt or black jack")
+            self.state = False
```

Comment:
I know that's not something you added with this PR, but I don't like this boolean state attribute, what is it supposed to mean?
"How are you?"
"I'm False!"
"Bruh, feel bad for you..."

So if you have an idea about this...

Reply:
I added that part and I agree that the naming is weird :upside_down_face: 
 I think we can rename `self.state` into `self.playing` or `self.playing_turn` ?